tests/testthat/test-dplyr-select.R

```diff
@@ -12,7 +12,6 @@ test_that("Empty select returns no columns", {
 })
 
 test_that("Empty select still includes the group_by columns", {
-  skip("add back in missing col: https://github.com/voltrondata/substrait-r/issues/136")
```

Comment:
Done!  

Can you clarify the circumstances in which we should add tests to test-pkg-dplyr.R instead of just the dplyr-verb-specific test files?  I'm not totally clear on that.

Reply:
I don't 100% get this, but perhaps let's discuss this when it comes up again